site/docs/v1/tech/email-templates/index.adoc

```diff
@@ -36,6 +36,27 @@ Here are the topics in this section:
 
 * link:/docs/v1/tech/email-templates/message-templates/[Message Templates]
 
+== Common Providers
+
+=== SendGrid
+
+[NOTE]
+====
+This information is for starting guidance only - FusionAuth connects seamlessly with many different email services. A deep exploration of email best practices and providers is outside the scope of this section.
+====
+
+SendGrid is a commonly used email provider and integrates well with FusionAuth via API. To setup SendGrid, please follow the instructions below:
```

Comment:
Not sure we should say "API" here - we don't use the SendGrid API - we use SMTP, and our config can be done via API or UI. I suppose you could say that SMTP is an API, but SendGrid also has an API, and we only integrate via SMTP.

Reply:
Good point!